.github/workflows/actions.yaml

```diff
@@ -5,7 +5,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.16.x, 1.17.x]
+        go-version: [1.17.x, 1.18.x, 1.19.x]
```

Comment:
I've not removed 1.17 as it's a required check, so needs to be present until we change it